tracing-error/src/lib.rs

```diff
@@ -16,20 +16,29 @@
 //!
 //! * [`ErrorLayer`], a [subscriber layer] which enables capturing `SpanTrace`s
 //!
+//! * [`InstrumentResult`] and [`InstrumentError`] extension traits, which
+//! provide an [`in_current_span()`] method for bundling errors with a
+//! [`SpanTrace`].
+//!
+//! * [`ExtractSpanTrace`] extension trait, for extracting `SpanTrace`s from
+//! behind `dyn Error` trait objects.
+//!
 //! **Note**: This crate is currently experimental.
 //!
 //! *Compiler support: requires `rustc` 1.39+*
 //!
-//! ## Usage
+//! ## Feature Flags
 //!
-//! Currently, `tracing-error` provides the [`SpanTrace`] type, which captures
-//! the current `tracing` span context when it is constructed and allows it to
-//! be displayed at a later time.
+//! - `stack-error` - Enables an experimental version of [`TracedError`] that
+//! statically wraps an error type without using Heap Allocations. The contained
+//! [`SpanTrace`] can still extract its SpanTrace from behind a
+//! [`std::error::Error`] trait object via type erasure.
```

Comment:
I don't know if "via type erasure" is accurate here..."type erasure" is what happens when a `T: Error` is _turned into_ a `dyn Error`...

Reply:
I figured type erasure was whenever you lose the information about the original type, going from `&TracedError<E>` to `&TracedError<()>` seems to fit the bill.

Reply:
hmm, that's true. i'm still not totally sure if this is really something we need to mention in the docs _here_ — it kind of feels like an implementation detail — but i'll defer to your judgement.